### PR TITLE
File Storage offline

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/Exceptions/1390290029.rst [hunk 1]
```diff
@@ -18,6 +18,13 @@ to create and save preview images. The :file:`processed` folder was set to a
 second file storage whose root directory was `readonly`. After fixing that
 permission problem of the second file storage, everything worked again.
 
+TYPO3 11.4 - File storage is offline / not found
+-------------------------------------------------
+
+Got to the backend module :guilabel:`File > Filelist` and check if a file
+storage is offline. If so right click on it and chose :guilabel:`edit`from
```
Comment (mbrodala) [inline Documentation/Exceptions/1390290029.rst:25]:
```suggestion
storage is offline. If so right click on it and chose :guilabel:`Edit` from
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/Exceptions/1390290029.rst [hunk 2]
```diff
@@ -18,6 +18,13 @@ to create and save preview images. The :file:`processed` folder was set to a
 second file storage whose root directory was `readonly`. After fixing that
 permission problem of the second file storage, everything worked again.
 
+TYPO3 11.4 - File storage is offline / not found
```
Comment (brotkrueml) [inline Documentation/Exceptions/1390290029.rst:21]: Really 11.4? Or do you mean 11.5 or 12.4?

File: Documentation/Exceptions/1390290029.rst [hunk 3]
```diff
@@ -18,6 +18,13 @@ to create and save preview images. The :file:`processed` folder was set to a
...
+-------------------------------------------------
+
+Got to the backend module :guilabel:`File > Filelist` and check if a file
+storage is offline. If so right click on it and chose :guilabel:`Edit` from
```
Comment (brotkrueml) [inline Documentation/Exceptions/1390290029.rst:25]:
```suggestion
storage is offline. If so right click on it and choose :guilabel:`Edit` from
```

File: Documentation/Exceptions/1390290029.rst (hunk 2 above)
Comment (linawolf) [inline Documentation/Exceptions/1390290029.rst:21]:
```suggestion
TYPO3 11.5 - File storage is offline / not found
```

Comment (linawolf) [reply Documentation/Exceptions/1390290029.rst]: woops 11.5

**Review by mbrodala — APPROVED** [review]

**Pull request merged**